File: app/src/main/java/org/phenoapps/intercross/AdapterEntry.java
package org.phenoapps.intercross;

import android.widget.Adapter;

public class AdapterEntry {
    public String crossId;
    public String timestamp;
    public String crossName;
    AdapterEntry(String cross, String timestamp, String crossName) {
        this.crossId = cross;
        this.timestamp = timestamp;
        this.crossName = crossName;
    }
}

File: app/src/main/java/org/phenoapps/intercross/AdapterEntry.kt
package org.phenoapps.intercross

class AdapterEntry {

    var first: String = String()

    var second: String = String()

    var third: String = String()

    var completed: Boolean = false

    var id: Int = -1

    constructor()

    constructor(first: String, second: String) {
        this.first = first
        this.second = second
    }

    constructor(first: String) {
        this.first = first
    }

    constructor(s: String, s1: String, s2: String) {
        this.first = s
        this.second = s1
        this.third = s2
    }

}